clio-transfer-model/src/main/scala/org/broadinstitute/clio/transfer/model/wgscram/WgsCramMetadata.scala

```diff
@@ -4,8 +4,19 @@ import java.net.URI
 import java.time.OffsetDateTime
 import java.util.UUID
 
+import io.circe.Json
+import io.circe.syntax._
 import org.broadinstitute.clio.transfer.model.{DeliverableMetadata, Metadata}
-import org.broadinstitute.clio.util.model.{DocumentStatus, RegulatoryDesignation}
+import org.broadinstitute.clio.util.json.ModelAutoDerivation
+import org.broadinstitute.clio.util.model.{
+  DataType,
+  DocumentStatus,
+  RegulatoryDesignation
+}
+
+object WgsCramMetadata extends ModelAutoDerivation {
+  val defaults: Json = WgsCramMetadata(dataType = Option(DataType.WGS)).asJson
```

Comment:
Am not convinced by this either. Having it here with the other client <-> server API stuff is pretty wide-open, and it's easy to think these defaults should be used in spots other than just recovery. Scoping the code to the only spot that it should be used (in the server backend) prevents abuse.

Reply:
I'm concerned that putting it in `ElasticSearchIndex` muddles the definition of the data with the actual values of the data. `ElasticSearchIndex` just defines the indices and doesn't seem like the place to store values.

Reply:
You could make the same argument about `TransferIndex` / `XYZMetadata`. Ideally they'd all just be declarations of the data's shape and not include any real values, but we need a spot to shim in the defaults. I'd rather keep the shimming as isolated as possible.